CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Guidelines for upgrading or adding new extensions
+
+One of the common worklow breakages caused by new extensions, or upgraded extensions, is failures in logical restores. A few workflows to verify when making changes to extensions are essentially various combinations straddling the change:
+
+- [ ] backup/restore on a vanilla project
```

Comment:
i am not sure that it is so much of a difference rn, but it can be a bigger thing in the future so i would add explicitly 

```suggestion
- [ ] backup/restore on a vanilla project
- [ ] pause/restore a vanilla project
```

Reply:
If there becomes a difference between the two, we'll need to think about it at that point. Right now there's no difference from a PG perspective.